dm/tests/sync_collation/conf/dm-task.yaml

```diff
@@ -0,0 +1,49 @@
+---
+name: sync_collation
+task-mode: all
+is-sharding: false
+meta-schema: "dm_meta"
+# enable-heartbeat: true
```

Comment:
heartbeat is deletted so  these conf is not required

Reply:
ok, i will delete.